frame/referenda/src/migration.rs

```diff
@@ -82,6 +82,10 @@ pub mod v0 {
 	#[storage_alias]
 	pub type ReferendumInfoFor<T: Config<I>, I: 'static> =
 		StorageMap<Pallet<T, I>, Blake2_128Concat, ReferendumIndex, ReferendumInfoOf<T, I>>;
+
+	#[storage_alias]
+	pub type ReferendumCount<T: Config<I>, I: 'static> =
```

Comment:
this item not used, below you use the storage from the pallet.

Reply:
Yes, I initially used this but forgot to remove it later.